build_tools/benchmarks/comparisons/mobilebert_int8_commands.py

```diff
@@ -0,0 +1,190 @@
+_DEFAULT_NUM_THREADS = 1
+
+
+class TfliteMobilebertInt8(TFLiteBenchmarkCommand):
+  """ Specializes the benchmark command to use TFLite. """
+
+  def __init__(self,
+               benchmark_binary: str,
+               model_name: str,
+               model_path: str,
+               test_data_dir: str,
+               driver: str = "cpu",
+               num_threads: int = _DEFAULT_NUM_THREADS,
+               num_runs: int = _DEFAULT_NUM_BENCHMARK_RUNS,
+               taskset: Optional[str] = None):
+    super().__init__(benchmark_binary,
+                     model_name,
+                     model_path,
+                     num_threads,
+                     num_runs,
+                     taskset=taskset)
+    self.driver = driver
+    self.args.append("--input_layer=input_ids,segment_ids,input_mask")
+    self.args.append("--input_layer_value_files=input_ids:" + test_data_dir +
+                     "/input_word_id.bin,segment_ids:" + test_data_dir +
+                     "/input_type_id.bin,input_mask:" + test_data_dir +
+                     "/input_mask.bin")
+    self.args.append("--input_layer_shape=1,384:1,384:1,384")
+
+
+class IreeMobilebertInt8(IreeBenchmarkCommand):
+  """ Specializes the benchmark command to use IREE. """
+
+  def __init__(self,
+               benchmark_binary: str,
+               model_name: str,
+               model_path: str,
+               driver: str = "local-task",
+               num_threads: int = _DEFAULT_NUM_THREADS,
+               num_runs: int = _DEFAULT_NUM_BENCHMARK_RUNS,
+               taskset: Optional[str] = None):
+    super().__init__(benchmark_binary,
+                     model_name,
+                     model_path,
+                     num_threads,
+                     num_runs,
+                     taskset=taskset)
+    self.driver = driver
+    self.args.append("--entry_function=main")
+    self.args.append(
+        '--function_input="1x384xi32=101 2129 2116 19576 2015 2106 3854 4679 2486 1029 102 1996 14169 2165 2019 2220 2599 1999 3565 4605 2753 1998 2196 11145 1012 8446 2001 3132 2011 7573 1005 1055 3639 1010 2029 14159 2032 2698 2335 1998 3140 2032 2046 2093 20991 2015 1010 2164 1037 19576 2029 2027 6757 2005 1037 7921 1012 7573 15674 3854 4679 2001 2315 3565 4605 12041 1010 3405 2274 3948 10455 1010 1016 13714 14918 1010 1998 2048 3140 19576 2015 1012 102 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0"'
```

Comment:
Is this reason why simple benchmark one can't be used? (not blocker, but just wondering if can make easier even :))

Reply:
Good point, ideally mobilebert_int8_commands.py and mobilebert_fp32_commands.py would re-use simple_commands.py. It does involve some refactoring since the mobilebert versions include input data but simple_commands doesn't support that atm. Let's keep this in mind when moving to the new benchmark infra that Che-Yu is building.